Rawl_B502/alia_peritia_de_genelogiis_&_de_generibus_hibernensium.ttl

```diff
@@ -880,12 +880,12 @@ _:missing-2d466a7a
 <#ConallCernach>
     a foaf:Person ;
     irishRel:nomName "Conall Cernach" ;
-    owl:sameAs <http://example.com/LL/laigsi.ttl#ConaillCernaig>, <http://example.com/LL/dimittamus_in_terim.ttl#ConaillCernaig>, <http://example.com/LL/dail_araide.ttl#ConaillCernaig>, <http://example.com/LL/dáil_araide.ttl#ConaillCernaig>, <http://example.com/LL/lagin.ttl#ConaillCernaig>, <http://example.com/Rawl_B502/do_forslointib_ulad_iar_coitchiund_in_so.ttl#ConallCernach>, <http://example.com/Rawl_B502/genelach_úa_n_echach.ttl#ConaillCernaig>, <http://example.com/Rawl_B502/mínugud_senchusa_laigin_and_so_sís.ttl#ConaillCernaig>, <http://example.com/Rawl_B502/na_ríg_immorro_iar_conchobor.ttl#ConaillCernaich>.
+    owl:sameAs <http://example.com/LL/laigsi.ttl#ConaillCernaig>, <http://example.com/LL/dimittamus_in_terim.ttl#ConaillCernaig>, <http://example.com/LL/dail_araide.ttl#ConaillCernaig>, <http://example.com/LL/dáil_araide.ttl#ConaillCernaig>, <http://example.com/LL/lagin.ttl#ConaillCernaig>, <http://example.com/Rawl_B502/do_forslointib_ulad_iar_coitchiund_in_so.ttl#ConallCernach>, <http://example.com/Rawl_B502/genelach_úa_n_echach.ttl#ConaillCernaig>, <http://example.com/Rawl_B502/mínugud_senchusa_laigin_and_so_sís.ttl#ConaillCernaig>, <http://example.com/Rawl_B502/na_ríg_immorro_iar_conchobor.ttl#ConaillCernaich>;
+    irishRel:ancestorOfGroup <#Conaille>.
 
 <#Conaille>
-    a foaf:Person ;
-    irishRel:nomName "Conaille" ;
-    rel:childOf <#ConallCernach>.
+    a irishRel:PopulationGroup ;
+    irishRel:populationGroupName "Conaille" .
```

Comment:
We need to make the equivalent change at line 885 of Rawl_B502\alia_peritia_de_genelogiis_&_de_generibus_hibernensium.ttl and make the two occurrences of the Conaille population group sameAs each other.

Reply:
I'm slightly confused.  This change is at line 885 so I am not sure where the other occurrence is that you mention.  I checked LL and didn't find any unless you mean <#ClannaConaillCernaig> in LL/dimittamus_in_terim.ttl